src/mmw/js/src/account/views.js

```diff
@@ -0,0 +1,76 @@
+"use strict";
+
+var Marionette = require('../../shim/backbone.marionette'),
+    models = require('./models'),
+    containerTmpl = require('./templates/container.html'),
+    profileTmpl = require('./templates/profile.html'),
+    accountTmpl = require('./templates/account.html');
+
+var ProfileView = Marionette.ItemView.extend({
+    template: profileTmpl
+});
+
+var AccountView = Marionette.ItemView.extend({
+    template: accountTmpl
+});
+
+var AccountContainerView = Marionette.LayoutView.extend({
+    // model AccountContainerModel
+
+    template: containerTmpl,
+
+    ui: {
+        profile: '[data-action="viewprofile"]',
+        account: '[data-action="viewaccount"]'
+    },
+
+    events: {
+        'click @ui.profile': 'viewProfile',
+        'click @ui.account': 'viewAccount'
+    },
+
+    modelEvents: {
+        'change:active_page': 'onChangeActivePage'
```

Comment:
This can be made a little simpler by:

```diff
diff --git a/src/mmw/js/src/account/views.js b/src/mmw/js/src/account/views.js
index 21a6ca2..63b9182 100644
--- a/src/mmw/js/src/account/views.js
+++ b/src/mmw/js/src/account/views.js
@@ -30,18 +30,13 @@ var AccountContainerView = Marionette.LayoutView.extend({
     },
 
     modelEvents: {
-        'change:active_page': 'onChangeActivePage'
+        'change:active_page': 'render'
     },
 
     regions: {
         infoContainer: '.account-page-container'
     },
 
-    onChangeActivePage: function() {
-        this.render();
-        this.showActivePage();
-    },
-
     showActivePage: function() {
         var activePage = this.model.get('active_page');
 
@@ -58,7 +53,7 @@ var AccountContainerView = Marionette.LayoutView.extend({
         }
     },
 
-    onShow: function() {
+    onRender: function() {
         this.showActivePage();
     },
 
```

Reply:
That was silly of me. I forced push this change.

Reply:
No, not silly. You were probably planning on changing `onChangeActivePage` in the future, as this feature fills out, so doing it that way would've made sense and was quite wise. However, in the spirit of not overengineering, we should do the simplest thing for the moment, and expand when prompted.